app/konnect/updates.md

```diff
@@ -37,6 +37,17 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 : The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin, and consumer are presented on the Overview Pages to nudge users to immediately dive into Konnect for the first time.
 
+**{{site.mesh_product_name}} in {{site.konnect_short_name}}**
+: {{site.mesh_product_name}} joins {{site.base_gateway}} and {{site.kic_product_name}} in {{site.konnect_short_name}} as a generally available control plane that is managed centrally by {{site.konnect_short_name}}’s management plane. This unified API platform allows you to see all configuration management and manage services from {{site.base_gateway}}, {{site.kic_product_name}}, and {{site.mesh_product_name}} in a single location. Mesh Manager also allows your organization to lower their operational overhead and elevate their developers’ experience by streamlining the operation of {{site.base_gateway}}s, {{site.kic_product_name}}, and {{site.mesh_product_name}} with a hosted control plane that can be implemented across any cloud, virtual machine, or on-premises solution.
+
+: You can use [Mesh Manager](/konnect/mesh-manager/) to create, modify, and delete mesh global control planes. Mesh global control planes are control planes that are mapped to different environments or business units. Each mesh global control plane maps to a Kubernetes or Linux zone through a process that is similar to adding a control plane for {{site.base_gateway}} using Gateway Manager. You can also now identify [service meshes](/mesh/latest/introduction/about-service-meshes/), view relevant metadata like meshes and zones, and access a service's details in {{site.konnect_short_name}}.
+
+: Mesh Manager is available now to Enterprise and new {{site.konnect_short_name}} customers:
+  * Enterprise customers will be able to access Mesh Manager immediately and can contact their sales representative for more information.
+  * New customers can apply their trial credits towards mesh zones.
+    For more information, see [{{site.konnect_short_name}} pricing](https://konghq.com/pricing).
+    Mesh Manager is not available for existing plus customers. Stay tuned for more information.
```

Comment:
@johnharris85 Could you review the wording of this to make sure this is accurate/everything we want to say for the changelog?